packages/cli/src/config/schema.ts

```diff
@@ -497,6 +497,13 @@ export const schema = {
 			default: 'dev',
 			env: 'N8N_RELEASE_TYPE',
 		},
+
+		gracefulShutdownTimeoutInS: {
```

Comment:
Nit: `InS` → I don't see any other env vars specifying the time unit in the name. Perhaps for consistency we should stick to this.

Reply:
Makes sense, tho I would like us to use suffixes when the value has a unit and it's not obviously clear what it is. Removed the suffix.